client/src/routes/game/planets/id/mod.rs

```diff
@@ -23,12 +26,9 @@ pub fn PlanetIDPage() -> impl IntoView {
     let id = move || params.with(|params| params.get("id").cloned().unwrap_or_default());
 
     let planet: Signal<Option<Planet>> = Signal::derive(move || {
-        game_wrapper.with(|state| {
-            console_log(format!("id: {} planets: {:?}", &id(), state.planets).as_str());
-            match state.planets.clone().get(&id()) {
-                Some(planet) => Some(planet.clone()),
-                None => None,
-            }
+        game_wrapper.with(|state| match state.planets.clone().get(&id()) {
+            Some(planet) => Some(planet.clone()),
```

Comment:
If you match an Option to return an Option, use `cloned()` instead
`game_wrapper.with(|state| match state.planets.get(&id()).cloned())`